llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -952,6 +957,44 @@ class StoreInst final : public Instruction {
 #endif
 };
 
+class UnreachableInst final : public Instruction {
+  /// Use UnreachableInst::create() instead of calling the constructor.
+  UnreachableInst(llvm::Instruction *I, Context &Ctx)
+      : Instruction(ClassID::Unreachable, Opcode::Unreachable, I, Ctx) {}
+  UnreachableInst(ClassID SubclassID, llvm::Instruction *I, Context &Ctx)
```

Comment:
Do we really need a second constructor in Unreachablelnst for allowing specific ClassID?

Reply:
No we don't need this constructor.

Reply:
removed it, thanks.